src/libs/sensors-logging.ts

```diff
@@ -392,24 +461,52 @@ PlayResY: ${videoHeight}
+Style: CenterTop,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,8,10,10,10,1
+Style: RightTop,Arial,${this.telemetryDisplayOptions.value.fontSize},${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontColor)},&H000000FF,${convertRGBToBGRColor(this.telemetryDisplayOptions.value.fontOutlineColor)},&H00000000,${this.telemetryDisplayOptions.value.fontBold ? '1' : '0'},${this.telemetryDisplayOptions.value.fontItalic ? '1' : '0'},${this.telemetryDisplayOptions.value.fontUnderline ? '1' : '0'},${this.telemetryDisplayOptions.value.fontStrikeout ? '1' : '0'},100,100,0,0,1,2,2,9,10,10,10,1
 
 [Events]
 Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text`
 
-    // Try to show, in order, variables the user has decided to show, or variables being used in the application, or all variables.
-    const hasUserSelected = !this.selectedVariablesToShow.value.isEmpty()
-    const areThereVariablesBeingUsed = !this.variablesBeingUsed.isEmpty()
-    const allAvailableVariables = Object.values(DatalogVariable)
-    const variablesToShow = hasUserSelected ? this.selectedVariablesToShow.value : areThereVariablesBeingUsed ? this.variablesBeingUsed : allAvailableVariables
+    /**
+     * Converts a hex color in the format #AARRGGBB to the .ass subtitle format &HAABBGGRR.
+     * @param {string} hexColor The hex color string in #AARRGGBB format.
+     * @returns {string} The color string in &HAABBGGRR format for .ass files.
+     */
+    function convertRGBToBGRColor(hexColor: string): string {
+      const red = hexColor.substring(1, 3)
+      const green = hexColor.substring(3, 5)
+      const blue = hexColor.substring(5, 7)
+      const alpha = hexColor.substring(7, 9) || 'FF'
+      const invertedAlpha = (255 - parseInt(alpha, 16)).toString(16).padStart(2, '0').toUpperCase()
+
+      return `&H${invertedAlpha}${blue}${green}${red}`
+    }
 
     log.forEach((logPoint, index) => {
       // Don't deal with the last log point, as it has no next point to compare to
       if (index === log.length - 1) return
 
-      const data = Object.entries(logPoint.data)
-        .filter((vData) => variablesToShow.includes(vData[0] as DatalogVariable))
-        .map((v) => ({ name: v[0], value: v[1].value }))
+      // Structured data object based on the user's telemetry display configuration
+      const data = Object.keys(this.telemetryDisplayData.value).reduce<Record<string, string>>((acc, quadrant) => {
+        acc[quadrant] = this.telemetryDisplayData.value[quadrant]
```

Comment:
A "quadrant" is 1/4, which makes this confusing. If you want to indicate the number of sections you can use "nonant" (1/9), or just use a more general word like "section"/"region"/"area"/"sector" :-)

Also, what is `acc` supposed to be short for?

Reply:
Your'e right. Changing to gridPosition